quizbizz/src/pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Navbar from "../../components/Navbar/Navbar";
+import Sidebar from "../../components/Sidebar/Sidebar";
+import AcheivementCategorySection from "./AcheivementCategorySection/AcheivementCategorySection";
+import HeroSection from "./HeroSection/HeroSection";
```

Comment:
keep folder name or everything in lowercase except components! apply this for all

Reply:
Done